content/articles/different-techniques-of-encoding-data-for-transmission/index.md

```diff
@@ -0,0 +1,101 @@
+### Introduction
+In a communication network, data is transmitted from the sender to the receiver. For this data to be transmitted properly it is encoded into binary digits 0s and 1s.
+
+This article intends to cover more on how data to be transmitted through a network is encoded. This therefore will emphasize more on different data encoding techniques. It will also talk about on digital and analog data.
+
+The different data encoding techniques that shall be focused on include; Non-Return to Zero Level, Non-Return to Zero, Non-Return to Zero Inverted, Bi-phase encoding, and Block encoding.
+
+Data encoding types shall also be discussed.
+For this data to be encoded it has to be converted to signal variations. This signal variations include;
```

Comment:
Make sure to format your intro section to improve readability. It seems to be technical for beginners.
Ping me when done :+1: